### PR TITLE
[Bugfix] qwen2.5-vl spec decode accept_len low

### PR DESCRIPTION
**Pull request by Lzhang-hub**
## Motivation

After this [pr](https://github.com/sgl-project/sglang/pull/13075), qwen2.5-vl eagle3 infer draft accept_len is low.

Because `aux_hidden_states` is miss in logits_processor.

Releated issue: https://github.com/sgl-project/SpecForge/issues/310

## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.ai/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.ai/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.ai/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.ai/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.ai/developer_guide/contribution_guide.html#benchmark-the-speed).
- [ ] Follow the SGLang code style [guidance](https://docs.sglang.ai/developer_guide/contribution_guide.html#code-style-guidance).
- [ ] Work with maintainers to merge your PR. See the [PR Merge Process](https://github.com/sgl-project/sglang/blob/main/.github/MAINTAINER.md#pull-request-merge-process)

### CODE REVIEW

**Review by yuan-luo — APPROVED** [review]
Thanks for the fix. LGTM.

**Review by JustinTong0323 — APPROVED** [review]

**Review by yhyang201 — APPROVED** [review]

**Comment by yhyang201** [comment]
@mickqian I think this is ready to be merged. Could you please merge it?
